gneiss/regression/_ols.py

```diff
@@ -203,11 +210,24 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
 
-    def fit(self, **kwargs):
-        """ Fit the model """
+    def fit(self, regularized=False, **kwargs):
+        """ Fit the model.
+
+        Parameters
+        ----------
+        regularized : bool
+            Specifies if a regularization procedure should be used
+            when performing the fit. (default = False)
+        **kwargs : dict
+           Keyword arguments used to tune the parameter estimation.
+
+        """
         for s in self.submodels:
             # assumes that the underlying submodels have implemented `fit`.
-            m = s.fit(**kwargs)
+            if regularized:
+                m = s.fit(**kwargs)
```

Comment:
This is backwards.  This should be `fit_regularized`,  not `fit`.  I'm removing the `fit_regularized` functionality for the time being to resolve this.  But this actually messed up the `test_mse` test.  I have fixed that in the upcoming commit